### PR TITLE
Fix loggers

### PR DESCRIPTION
**Pull request by Ladsgroup**
1- Moving log level to warning
2- Removing filters and using loggers instead
3- Removing logger settings from python files, except for the uwsgi itself

### CODE REVIEW

**Comment by Ladsgroup** [comment]
Untested, I test it very soon :)

**Comment by Ladsgroup** [comment]
About the second one: [See this](https://www.python.org/dev/peps/pep-0391/) it's not an official PEP but it is very helpful and based the official manual of logging.config

**Comment by Ladsgroup** [comment]
Tested, works really good

File: ores_celery.py [hunk 1]
```diff
@@ -22,5 +22,4 @@
 application = score_processor.application
 
 if __name__ == '__main__':
-    logging.getLogger('ores').setLevel(logging.DEBUG)
-    application.worker_main(argv=["celery_worker", "--loglevel=INFO"])
+    application.worker_main(argv=["celery_worker"])
```
Comment (halfak) [inline ores_celery.py]: By setting the log level up in **main**, we can see more debugging when running tests.  Why get rid of that?

Comment (Ladsgroup) [inline ores_celery.py]: Good point. Fixing it right away

**Pull request merged**